src/main/java/seedu/address/ui/MainWindow.java

```diff
@@ -188,6 +189,22 @@ private void handleExit() {
         primaryStage.hide();
     }
 
+    /**
+     * Handles the display of Command History in the GUI.
+     */
+    private void handleHistory() {
```

Comment:
The printing out on the console is for debugging purposes right now. Because i needed to verify the interaction between the UI and the logic in ModelHistoryManager. 

The getUndoCommandHistory and getRedoCommandHistory are part of the previous implementation (before you implemented the CommandRecord object, when I was returning a List of Strings). I will be deprecating these in a future update, as these methods will still remain useful for debugging for this feature. Once we've verified the feature works well, I'll proceed to deprecate the methods.

Reply:
Ahh I see, if I understand it correctly, actually, i think you don't have to deprecate this, just log this in logger instead :)